config/configuration.go

```diff
@@ -177,10 +177,11 @@ func ValidateUrl(c *cli.Context, allowURLFromArgs bool) (*url.URL, error) {
 }
 
 type UnvalidatedIngressRule struct {
-	Hostname      string              `json:"hostname,omitempty"`
-	Path          string              `json:"path,omitempty"`
-	Service       string              `json:"service,omitempty"`
-	OriginRequest OriginRequestConfig `yaml:"originRequest" json:"originRequest"`
+	Hostname        string              `json:"hostname,omitempty"`
+	Path            string              `json:"path,omitempty"`
+	PathReplacement string              `yaml:"pathReplacement" json:"pathReplacement,omitempty"`
```

Comment:
Could we rename this to something like `Rewrite`? I think that would make it more clear that a rewrite is happening on the path before the proxied request is made to the origin.

Reply:
That sounds good, as it will follow naming similar to NGINX. Done.